### PR TITLE
Add more clarification to k8s ASG docs

### PR DESCRIPTION
**Pull request by harsh-px**
Signed-off-by: Harsh Desai <harsh@portworx.com>

### CODE REVIEW

File: _includes/asg/ebs-template.md [hunk 1]
```diff
@@ -10,21 +14,26 @@ Ensure that these EBS volumes are created in the same region as the auto scaling
 
 Record the EBS volume ID (e.g. _vol-04e2283f1925ec9ee_), this will be passed in to PX as a parameter as a storage device.
 
-**For PX version > 1.3**
+### Using a template specification
 
-A set of EBS volume templates can be specified in the following comma-separated format
+For PX 1.3 and higher, you can specify a template spec which will be used by Portworx to create new EBS volumes.
```
Comment (adityadani) [inline _includes/asg/ebs-template.md]: Can we make it explicit in UI as well that this is supported only for 1.3

File: _includes/asg/ebs-template.md [hunk 2]
```diff
@@ -10,21 +14,26 @@ Ensure that these EBS volumes are created in the same region as the auto scaling
...
+"type=<EBS volume type>,size=<size of EBS volume>,iops=<IOPS value>"
 ```
-"type=<EBS volume type>,size=<size of EBS volume>,iops=<IOPS value>","type=<EBS volume type>,size=<size of EBS volume>,iops=<IOPS value>"
 
-For ex:
+* __type__: Following two types are supported
+    * _gp2_
+    * _io1_ (For io1 volumes specifying the iops value is mandatory.)
+* __size__: This is the size of the EBS volume in GB
+* __iops__: This is the required IOs per second from the created EBS volume.
```
Comment (adityadani) [inline _includes/asg/ebs-template.md]: nit: This is the required IOs per second from the EBS volume.

File: _includes/asg/ebs-template.md [hunk 3]
```diff
@@ -10,21 +14,26 @@ Ensure that these EBS volumes are created in the same region as the auto scaling
...
 
-"type=gp2,size=100","type=io1,size=200,iops=1000"
+See [EBS details](https://aws.amazon.com/ebs/details/) for more details on above parameters.
 
-```
+### Examples
```
Comment (adityadani) [inline _includes/asg/ebs-template.md]: nit: I think this should be a sub header to "template specification" header.

File: cloud/aws/kops-asg.md [hunk 4]
```diff
@@ -44,7 +50,7 @@ Portworx gets deployed as a [Kubernetes DaemonSet](https://kubernetes.io/docs/co
 When generating the spec, following parameters are important for KOPS:
 1. __AWS environment variables__: In the environment variables option (_e_), specify _AWS\_ACCESS\_KEY\_ID_ and _AWS\_SECRET\_ACCESS\_KEY_ for the KOPS IAM user. Example: AWS_ACCESS_KEY_ID=\<id>,AWS_SECRET_ACCESS_KEY=\<key>. If you are using instance privileges you can ignore setting the environment variables.
 
-2. __Volume template__: In the drives option (_s_), specific the EBS volume template that you created in previous step. Portworx will dynamically create EBS volumes based on this template.
+2. __Volume template__: In the drives option (_s_), specific the EBS volume template that you created in [previous step](#ebs-volume-template). Portworx will dynamically create EBS volumes based on this template.
```
Comment (adityadani) [inline cloud/aws/kops-asg.md]: nit: specify the EBS volume

File: scheduler/kubernetes/px-k8s-asg.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+%}
+
+## Install
+
+Portworx gets deployed as a [Kubernetes DaemonSet](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/). Following sections describe how to generate the spec files and apply them.
+
+### Generate the Portworx Spec
+
+When generating the spec, following parameters are important:
+1. __Volume template__: In the drives option (_s_), specific the EBS volume template that you created in [previous step](#ebs-volume-template). Portworx will dynamically create EBS volumes based on this template.
```
Comment (adityadani) [inline scheduler/kubernetes/px-k8s-asg.md]: nit: specify the EBS volume

File: _includes/asg/ebs-template.md (hunk 1 above)
Comment (harsh-px) [reply _includes/asg/ebs-template.md]: Removing it from the dublin spec: https://github.com/portworx/px-installer/pull/79

File: scheduler/kubernetes/px-k8s-asg.md (hunk 5 above)
Comment (harsh-px) [reply scheduler/kubernetes/px-k8s-asg.md]: done

File: cloud/aws/kops-asg.md (hunk 4 above)
Comment (harsh-px) [reply cloud/aws/kops-asg.md]: done

File: _includes/asg/ebs-template.md (hunk 3 above)
Comment (harsh-px) [reply _includes/asg/ebs-template.md]: done

File: _includes/asg/ebs-template.md (hunk 2 above)
Comment (harsh-px) [reply _includes/asg/ebs-template.md]: done

**Pull request merged**